modules/scripts.nix

```diff
@@ -138,5 +142,11 @@ in {
   '';
 
   # Processes
-  processes.entryscript.exec = "${entryScript}";
+  processes.entryscript.exec = ''
+    ${entryScript}
+  '';
+
+  scripts.caddy-trust.exec = ''
+    ${config.services.caddy.package}/bin/caddy trust
+  '';
```

Comment:
```suggestion
  scripts.caddy-trust.exec = ''
    ${config.services.caddy.package}/bin/caddy trust
  '';

  # Processes
  processes.entryscript.exec = ''
    ${entryScript}
  '';
```

Reply:
You may also move `# Processes`  I just couldn't mark it

Reply:
The Process needs to be defined where the script is defined :) so i cannot move the process.

Reply:
I meant below, so that the scripts are defined in a block 🙂